plugins/jobs/web_client/templates/JobList.pug

```diff
@@ -0,0 +1,27 @@
+.page-size-container
+  span(style="margin-right:10px") Page size
+  select.g-page-size
+    each val in pageSizes
+      option(selected=val==pageSize)= val
+
+.filter-container.clearfix
+  .type
+  .status
+
+ul.jobs.nav.nav-tabs
```

Comment:
For classes that are specific to Girder, let's preface them with `g-` to avoid name collision with third party library selectors.